app/routes.py

```diff
@@ -1,2 +1,96 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.planet import Planet
+from app import db
+
+planets_bp = Blueprint('planets', __name__, url_prefix='/planets')
+
+@planets_bp.route("", methods=['POST'])
+def create_planet():
+    request_body = request.get_json()
+    new_planet = Planet(
+        name = request_body["name"],
+        description = request_body["description"],
+        mass = request_body["mass"],
+    )
```

Comment:
This might be a good candidate for error handling because it is possible a name, description or mass is not included!